app/shared/src/popup/BarPopup.js

```diff
@@ -29,8 +43,16 @@ export default class BarPopup extends React.Component<Props> {
   render = () => (
     <Popup isVisible={this.props.isVisible} onClose={this.onClose}>
       <View style={styles.bar}>
-        <LinkButton title="Cancel" onPress={this.onClose} />
-        <LinkButton title={this.props.buttonTitle} onPress={this.onSave} />
+        <LinkButton
+          title="Cancel"
+          onPress={this.onClose}
+          styles={cancelLinkButtonStyles}
```

Comment:
Also, I would change "styles" => "style" to stick with the common naming: `<View style={styles.view}>`...

Reply:
LinkButton expects ButtonProps which is:
```
export type Props = {
  title: string,
  onPress?: Function,
  // $FlowFixMeProps
  styles?: {|
    buttonWrapper?: Object,
    button?: Object,
    buttonText?: Object,
    icon?: Object,
  |},
  icon?: Object,
};
```

Reply:
Oh yeah, `Button` component design is pretty shitty. I already started to work on refactoring. Alright, thanks for the explanation.